common/logger/src/diem_logger.rs

```diff
@@ -30,6 +30,7 @@ use std::{
 };
 
 const RUST_LOG: &str = "RUST_LOG";
+const RUST_LOG_REMOTE: &str = "RUST_LOG_REMOTE";
```

Comment:
Should we add documentation about how users are supposed to config this?  Or document it elsewhere?

Reply:
I mean I can add a comment here, but AFAIK there's very little actual documentation around usage of our software. In that we don't really have the equivalent of a `man` page.

Reply:
I think a comment here about how the environment variables should be used would be good.

Reply:
We can add to the documentation in the operator runbooks.  There is one for monitoring under full nodes.